packages/renderer/src/lib/markdown/Markdown.svelte

```diff
@@ -77,18 +98,26 @@ onMount(() => {
             e.target.disabled = false;
             e.target.firstChild.style.display = 'none';
           });
+      } else if (e.target instanceof HTMLAnchorElement) {
+        // Execute the command since it's a simple "link" to it
+        // usually associated with a dialog / quickpick action.
+        window.executeCommand(command);
```

Comment:
Should we display an error if the command fails somehow?

Reply:
No implementation so far for error reporting, but we do have error reporting within the command like the compose selection install.

What do you think? I see that we have the error reporting within the "button" for the other implementation, but unsure what we should do with this.

Task bar / notification section maybe?

Reply:
Here it's calling a promise no ? If yes probably need to handle catch of the error to avoid u handled exception